js/index.js

```diff
@@ -1,271 +1,89 @@
-    f8Board.addEventListener("click", function(event) {
-    });
-
-    // First column. 'g' column
-    const g1Board = document.getElementById("g1");
-    g1Board.addEventListener("click", function(event) {
-    });
-
-    const g2Board = document.getElementById("g2");
-    g2Board.addEventListener("click", function(event) {
-    });
-
-    const g3Board = document.getElementById("g3");
-    g3Board.addEventListener("click", function(event) {
-    });
-
-    const g4Board = document.getElementById("g4");
-    g4Board.addEventListener("click", function(event) {
-    });
-
-    const g5Board = document.getElementById("g5");
-    g5Board.addEventListener("click", function(event) {
-    });
-
-    const g6Board = document.getElementById("g6");
-    g6Board.addEventListener("click", function(event) {
-    });
-
-    const g7Board = document.getElementById("g7");
-    g7Board.addEventListener("click", function(event) {
-    });
+import {Board} from './board.js';
+const board = new Board();
+
+let activePiece = null; //global variable to keep track of active piece
+let currentPlayer = 0; //global variable to keep track of currentPlayer turn- set to white piece initially. 
+/* 
+Helper function to "reset" the colors of the board,
+and makes the colors of the squares black and white, 
+(if it's an even square, it's white; if it's odd, it's black)
+and removes the highlights. 
+*/
+function colorAllSquares() {
+    const rowStr = "87654321";
+    const colStr = "abcdefgh";
+    for (let i = 0; i < board.board.length; i++){
+        for (let j = 0; j < board.board[i].length; j++) {
+            const squarePos = colStr[j] + rowStr[i];
+            const evenSquare = (i + j) % 2 == 0;
+            const color = evenSquare ? "white" : "black";
+            const getSquare = document.getElementById(squarePos);
+            getSquare.classList.remove("bg-primaryRed", "bg-primaryRedBlack");
+            getSquare.classList.add("bg-" + color);
+        }
+    }
+}
 
-    const g8Board = document.getElementById("g8");
-    g8Board.addEventListener("click", function(event) {
-    });
 
-    // First column. 'h' column
-    const h1Board = document.getElementById("h1");
-    h1Board.addEventListener("click", function(event) {
-    });
+/*
+Function for event handlers that resets the colors of the chess squares,
+and then proceeds to mark the possible squares a chess piece can 
+move to if the piece is there. Active piece variable is then updated.
+*/
+function initBoardEvListener() {
+    const rows = 8;
+    const cols = ["a", "b", "c", "d", "e", "f", "g", "h"];
+
+    for (let i = 1; i <= rows; i++) {
+        for (let j = 0; j < cols.length; j++) {
+            const squareString = cols[j] + i.toString();
+            const element = document.getElementById(squareString);
+            element.addEventListener("click", () => {
+                const piece = getPiece(squareString);
+                if (piece && piece.pieceId == currentPlayer) {
```

Comment:
You'll want to use `.getPlayerId()` instead of `.pieceId` in the `if` statement on line `43` since `.pieceId` is not a valid attribute.